src/pages/boundary/api-docs/[[...page]].tsx

```diff
@@ -92,9 +106,19 @@ export async function getStaticProps({ params }) {
 	// Construct sidebar nav data levels
 	const sidebarNavDataLevels = [
 		{
+			backToLinkProps: { text: 'Boundary Documentation', href: '/boundary/' },
```

Comment:
This text should actually be `Boundary Home` like other non /docs documentation pages have it. I see that's not what in the designs, but that change is deferred until the IA updates RFC is approved and in motion.

```suggestion
			backToLinkProps: { text: 'Boundary Home', href: '/boundary/' },
```